user-service/src/main/java/rs/edu/raf/banka1/model/CurrentAccount.java

```diff
@@ -0,0 +1,25 @@
+package rs.edu.raf.banka1.model;
+
+import jakarta.persistence.*;
+import lombok.Getter;
+import lombok.Setter;
+
+@Entity
+@Getter
+@Setter
+public class CurrentAccount {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    private String accountNumber;
+    private Long ownerId;
+    private Double balance;
+    private Double availableBalance;
+    private Long createdByAgentId;
+    private Integer creationDate;
```

Comment:
I would change the type of attributes creationDate, expirationDate to Long if these values should be saved as Unix Timestamp.

Reply:
Solved.